deform/locale/fi/LC_MESSAGES/deform.po

```diff
@@ -0,0 +1,122 @@
+# Translations template for deform.
+# Copyright (C) 2013 ORGANIZATION
+# This file is distributed under the same license as the deform project.
+# Antti Haapala <antti.haapala@anttipatterns.com>, 2013.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: deform 2.0a2\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
```

Comment:
Hmm, it says `Report-Msgid-Bugs-To`, not `msgstr` :D (hadn't noticed that one even)